src/browser/services/simDriver.js

```diff
@@ -13,6 +13,7 @@ angular.module('Simulator').factory('SimDriver', function SimDriverService() {
 
     ctrl.setQueue = function(parsedData) {
         //mem queue template object
+        let returnQueue = [];
```

Comment:
Currently this setQueue doesn't have real logical effect since it gives it the same tag index offset for every address. It seems now that we've changed the purpose of file parser and the MAQ on the cache display screen we don't really need a queue that feeds into cache detail.

Reply:
I completely agree, I was just trying to get the table functionality setup for when we had the data available for the Cache details page.

Reply:
so do you wanna mark this as good then and make a new issue for actually connecting the dots? I'm cool with that if thats the route you wanna take. Ill approve and merge it if thats cool or fix it and request a review from me